packages/api/src/modules/providers/scdl/scdl.mapper.ts

```diff
@@ -47,7 +48,10 @@ export const SCDL_MAPPER: ScdlGrantSchema = {
     allocatorName: { path: [["nomAttribuant", "Nom de l'attribuant"]] },
     allocatorSiret: {
         path: [["idAttribuant", "Identification de l'attribuant (SIRET)"]],
-        adapter: v => v?.toString(),
+        adapter: v => {
+            const str = v?.toString();
+            if (str) return str.split(".")[0];
```

Comment:
Je suis étonnée qu'il y ait besoin de cette modif'. Même si dans un tableur c'est traité en flottant et affiché avec une virgule, je m'attends à ce toString garde une version entière non ? Et dans le cas contraire, est-ce qu'on ne veut pas plutôt forcer en entier *avant* de transformer en string ?

Reply:
Pour moi `v?.toString()` ne sert à rien puisque, sauf erreur de ma part, le parsing `csvSyncParser.parse()` met déjà tout en chaine de caractère, non ? Je l'ai laissé au cas ou quelque chose m'échappe mais c'est possible de ne garder que le split pour moi.
Quand il arrive "v" est déjà une string qui ressemble à ca "123456789.0". Le split me semble donc être la solution la plus résiliente.

Reply:
Alors le `.split` est pas bon car si demain on a `123456789,0` ça va péter (même si c'est peu probable, on ne peux s'assurer du format comme ça). Je ferai un cast en `Number` et un `toFixed(0)`. mais je suis d'accord avec Alice sur le fait que ce soit étrange qu'on reçoive ce format ici. Ceci dit vu que tu n'as rien changé avant...

Reply:
Alors après vérification j'avais pas noté que le format "123456789.0" est présent DANS le CSV. En fait c'est assez embêtant et si j'avais percuté ça plus tôt j'aurais demandé à retourner le jeu au fournisseur pour qu'il corrige ou pour comprendre l'intéret.

Comme quickfix ta solution du split me va @461OceanBd mais je veux un if (v.includes(".")) => doStuff avec un commentaire `// TODO: quickfix for Centre Val de Loire, remove when CSV will be cleaned`